rpcd/playbooks/roles/rpc_maas/templates/elasticsearch_process_check.yaml.j2

```diff
@@ -0,0 +1,19 @@
+type: agent.plugin
```

Comment:
Also link back to here with the bug created so we know this item was "resolved" to the level it needs to be here.